napi-inl.h

```diff
@@ -1628,6 +1628,10 @@ inline uint8_t TypedArray::ElementSize() const {
     case napi_float32_array:
       return 4;
     case napi_float64_array:
+#ifdef NAPI_EXPERIMENTAL
```

Comment:
N-API BigInt support has been out of experimental and can be guarded with
```
#if (NAPI_VERSION >= 6)
```

Check out https://github.com/nodejs/node-addon-api/pull/697 for more contexts.